driver/src/main/java/com/code_intelligence/jazzer/junit/FuzzTest.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 Code Intelligence GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.code_intelligence.jazzer.junit;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.condition.DisabledIfEnvironmentVariable;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.ResourceAccessMode;
+import org.junit.jupiter.api.parallel.ResourceLock;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+/**
+ * A {@link ParameterizedTest} with input data generated dynamically by the Java fuzzer <a
+ * href="https://github.com/CodeIntelligenceTesting/jazzer">Jazzer</a>.
+ *
+ * When executed as a JUnit Jupiter test, the fuzz test is executed on all files in the seed corpus
+ * (see documentation for the {@code seedCorpus} parameter.
+ *
+ * <p>Methods annotated with {@link FuzzTest} must take a single parameter of type {@code byte[]} or
+ * {@link com.code_intelligence.jazzer.api.FuzzedDataProvider}. The latter provides convenience
+ * methods to generate common Java types from the raw fuzzer input.
+ *
+ * <p>Only a single method per class can be annotated with {@link FuzzTest}.
```

Comment:
It will be enforced by the Jazzer engine, but it would be better if we could also enforce it on the JUnit level - I didn't find a good way to do that though.

I will take a stab at making multiple `@FuzzTest` work instead - shouldn't be that difficult.

Reply:
Was easier than expected, done in a follow-up commit. This will also simplify the Jazzer test engine design.